### PR TITLE
FAQ: Add systemd-repart.service masking entry

### PR DESCRIPTION
**Pull request by travier**

### CODE REVIEW

File: modules/ROOT/pages/faq.adoc [hunk 1]
```diff
@@ -448,3 +448,15 @@ If your system is in this state you have two options:
 * Re-deploy starting with the latest image artifacts.
 ** This means you start with the latest policy.
 * Follow the workaround in https://github.com/coreos/fedora-coreos-tracker/issues/701[coreos/fedora-coreos-tracker#701] to restore the base policy.
+
+== Why is the systemd-repart.service system unit masked?
+
+https://www.freedesktop.org/software/systemd/man/systemd-repart.html[system-repart]
+is tool to grow and add partitions to a partition table. On Fedora CoreOS, we
```
Comment (jlebon) [inline modules/ROOT/pages/faq.adoc:455]:
```suggestion
is a tool to grow and add partitions to a partition table. On Fedora CoreOS, we
```

File: modules/ROOT/pages/faq.adoc [hunk 2]
```diff
@@ -448,3 +448,15 @@ If your system is in this state you have two options:
...
+only support using Ignition to create partitions, filesystems and mount points,
+thus systemd-repart is masked by default.
+
+Ignition runs on first boot in the initramfs and is aware of Fedora CoreOS
+specific disk layout. It is also capable of reconfiguring the root filesystem
+(from xfs to ext4 for example), setting up LUKS, etc..See the
```
Comment (jlebon) [inline modules/ROOT/pages/faq.adoc:461]:
```suggestion
(from xfs to ext4 for example), setting up LUKS, etc... See the
```

**Review by jlebon — APPROVED** [review]
Two minor comments, LGTM otherwise!

File: modules/ROOT/pages/faq.adoc [hunk 3]
```diff
@@ -448,3 +448,15 @@ If your system is in this state you have two options:
 * Re-deploy starting with the latest image artifacts.
 ** This means you start with the latest policy.
 * Follow the workaround in https://github.com/coreos/fedora-coreos-tracker/issues/701[coreos/fedora-coreos-tracker#701] to restore the base policy.
+
+== Why is the systemd-repart.service system unit masked?
```
Comment (dustymabe) [inline modules/ROOT/pages/faq.adoc:452]:
```suggestion
== Why is the systemd-repart.service systemd unit masked?
```

maybe?

**Comment by dustymabe** [comment]
Not sure if we want to offer an example of how to unmask it like we do for `dnsmasq` or not. I'll leave that up to you.

**Review by dustymabe — APPROVED** [review]
LGTM

**Comment by travier** [comment]
> Not sure if we want to offer an example of how to unmask it like we do for `dnsmasq` or not. I'll leave that up to you.

Added a note referring to the one used for unmasking dnsmasq.

**Review by jlebon — APPROVED** [review]

**Pull request merged**